src/App.tsx

```diff
@@ -5,7 +5,22 @@ import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+
+  const addMovie = (newMovie: Movie) => {
+    if (movies) {
```

Comment:
The movies variable is an array and all arrays are truthy in JavaScript, so this condition is not necessary. Arrays, even if they are empty, are always truthy.

Reply:
Hello. Thanks for your attention about my mistake. Would you suggest - How to avoid this warning in typescript correctly.
At now, i've fixed codition to 
`if (movies.length > 0) {}`